esphome/components/debug/debug_esp32.cpp

```diff
@@ -276,6 +276,19 @@ void DebugComponent::get_device_info_(std::string &device_info) {
   device_info += " Cores:" + to_string(info.cores);
   device_info += " Revision:" + to_string(info.revision);
 
+  // Framework detection
+  device_info += "|Framework: ";
+#ifdef USE_ARDUINO
```

Comment:
```suggestion
#if defined(USE_ARDUINO)
```

Just for consistency with `#elif defined(USE_ESP_IDF)`. Not really important